core/services/relay/evm/functions/logpoller_wrapper.go

```diff
@@ -135,20 +129,16 @@ func (l *logPollerWrapper) LatestEvents() ([]evmRelayTypes.OracleRequest, []evmR
 		l.lggr.Debug("LatestEvents: no non-zero coordinators to check")
 		return resultsReq, resultsResp, errors.New("no non-zero coordinators to check")
 	}
-	if latest < nextBlock {
-		l.lggr.Debugw("LatestEvents: no new blocks to check", "latest", latest, "nextBlock", nextBlock)
-		return resultsReq, resultsResp, nil
-	}
 
 	for _, coordinator := range coordinators {
-		requestLogs, err := l.logPoller.Logs(nextBlock, latest, functions_coordinator.FunctionsCoordinatorOracleRequest{}.Topic(), coordinator)
+		requestLogs, err := l.logPoller.Logs(startBlock, latest, functions_coordinator.FunctionsCoordinatorOracleRequest{}.Topic(), coordinator)
```

Comment:
I don't like that we're relying on pluginORM.CreateRequest() to filter out duplicates:
- we will be returning potentially lots of data from LatestEvents() for no good reason.
- we lose the ability to differentiate from an actual buggy duplicate because you disabled a log in handleOracleRequestV1().

Can LogPollerWrapper still keep local state of returned IDs and not return the same one twice?